src/cell.rs

```diff
@@ -1,15 +1,15 @@
 //! Helper module for some internals, most users don't need to interact with it.
 
-#![allow(missing_docs)]
-
 use std::cell::UnsafeCell;
 use std::error::Error;
 use std::fmt::{Display, Error as FormatError, Formatter};
 use std::ops::{Deref, DerefMut};
 use std::sync::atomic::{AtomicUsize, Ordering};
+use std::usize;
 
+/// Marker struct for an invalid borrow error
 #[derive(Clone, Copy, Debug)]
-pub struct InvalidBorrow;
```

Comment:
This seems to be an implementation detail, since problems always result in panics.

Reply:
:thinking: Maybe we should add `try_*` variants?

Reply:
As you mentioned, that would be helpful for FFI

Reply:
I'll add the try variants :)